nltk/metrics/distance.py

```diff
@@ -34,7 +35,8 @@ def _edit_dist_init(len1, len2):
     return lev
 
 
-def _edit_dist_step(lev, i, j, s1, s2, substitution_cost=1, transpositions=False):
+def _edit_dist_step(lev, i, j, s1, s2,
+                    substitution_cost=1, transpositions=False):
```

Comment:
No reason to break this unless it's over the 80 char lens. Even so, the break should be one per kwargs, so 

```
def _edit_dist_step(lev, i, j, s1, s2,
                     substitution_cost=1, 
                       transpositions=False):
```

Reply:
Break the other keyword into a second line too, that'll be more consistent. Thanks!

Reply:
cool